backend/spec/features/admin/orders/shipments_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'spec_helper'
 
-describe "Shipments", type: :feature do
+describe "Shipments", type: :feature, js: true do
```

Comment:
Can we add this only to the new test addition? Tests that use javascript are slower, so it's better not to affect all of them.